src/DoctrineGui/Form/ClientFieldset.php

```diff
@@ -49,6 +49,20 @@ public function init()
         );
 
         $this->add([
+                'name'          => 'user',

```

Comment:
Do you mean like: 'name' => 'user', (i.e. a single space?) and not using padding would be: 'name'=>'user' ?